airline-web/app/controllers/ClientActor.scala

```diff
@@ -18,10 +25,37 @@ class ClientActor(out: ActorRef, chat: ActorRef) extends Actor {
       if (text.indexOf("[LOGGED]") > -1) {
 	   chat ! ClientSentMessage(text)
 	  } else {
-	  chat ! ClientSentMessage("[" + sdf.format(Calendar.getInstance().getTime())+ "] " + text)
+		  val json_text: JsValue = Json.parse(text)
+		  val airline = AirlineSource.loadAirlineById(userId)
```

Comment:
Merged the code in but this line is not quite right...airlineId is not the same as userId, it's throwing some exceptions in prod now

Not sure if we want to load the airline again here. Do we really have to?